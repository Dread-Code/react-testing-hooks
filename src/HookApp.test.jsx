import React from "react";
import { shallow } from "enzyme";
import { HookApp } from "./HookApp";
import "@testing-library/jest-dom";

describe("HookApp", () => {
  test("should match snapshoot", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
