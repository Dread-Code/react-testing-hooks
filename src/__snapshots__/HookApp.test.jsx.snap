// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookApp should match snapshoot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
